src/core/load-fragment.c

```diff
@@ -1697,6 +1697,8 @@ int config_parse_exec_cpu_affinity(
                 void *userdata) {
 
         ExecContext *c = data;
+        const Unit *u = userdata;
+        char *k;
```

Comment:
`k` is not freed. Please use `_cleanup_free_` attribute:
```
_cleanup_free_ chra *k = NULL;
```

Reply:
Ready, thank you